packages/renderer/src/lib/featured/FeaturedExtension.svelte

```diff
@@ -7,6 +7,7 @@ import FeaturedExtensionDownload from './FeaturedExtensionDownload.svelte';
 
 export let featuredExtension: FeaturedExtension;
 export let variant: 'primary' | 'secondary' = 'primary';
+export let title: string | undefined = undefined;
```

Comment:
AFAIK the title should be provided by this component, not by the client

so maybe displayTitle: boolean = false

Reply:
Fixed, I changed to an hardcoded text hidden by default